src/pages/Dummy/index.js

```diff
@@ -30,6 +64,21 @@ function Dummy(){
             <Link to={ROUTES.WORKOUTS}>Workouts</Link>
         </li>
     </ul>
+
+    <LolButton handleOnClick={handleLolButtonClick}/>
```

Comment:
handleLolButtonClick ? 

wtf. 
last time you call a component and handler with LOL